src/main/java/ru/skypro/homework/service/impl/AuthServiceImpl.java

```diff
@@ -16,23 +18,36 @@ public class AuthServiceImpl implements AuthService {
 
     private final UserDetailsManager manager;
     private final PasswordEncoder encoder;
+    private final CustomUserDetailsService customUserDetailsService;
+    private final UserRepository userRepository;
 
     static ArrayList<String> userLog = new ArrayList<>();
 
-    public AuthServiceImpl(UserDetailsManager manager,
-                           PasswordEncoder passwordEncoder) {
+    //public AuthServiceImpl(UserDetailsManager manager, PasswordEncoder encoder, UserRepository userRepository) {
+    //    this.manager = manager;
+    //    this.encoder = encoder;
+    //    this.userRepository = userRepository;
+    //}
+
+
+    public AuthServiceImpl(UserDetailsManager manager, PasswordEncoder encoder, CustomUserDetailsService customUserDetailsService, UserRepository userRepository) {
```

Comment:
```suggestion
    public AuthServiceImpl(UserDetailsManager manager, PasswordEncoder encoder, UserDetailsService customUserDetailsService, UserRepository userRepository) {
```
инъекцию надо делать всегда по интерфейсу